nvd3/templates/content.html

```diff
@@ -96,6 +96,12 @@
 
     {% endblock custoattr %}
 
+    {% block y_axis_scale %}
+    {% if chart.y_axis_scale_min or chart.y_axis_scale_max %}
```

Comment:
@areski I did not find any documentation on the forcceY method and experimented a bit with it and worked for me all the time. Also `[,1]` works flawless in case only the max value is defined.
I could add an if statement in the case that only min is defined, but for the max case the first parameter needs to stay undefined anyway.
It seems more consistent to me to leave it undefined in both cases.